cli/integrity/integrity.go

```diff
@@ -0,0 +1,25 @@
+package integrity
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// IntegritySigner is a noop integrity check preparation.
+type IntegritySigner struct{}
+
+// NewIntegritySigner is a noop constructor of an IntegritySigner based
+// on a private key file.
+func NewIntegritySigner(string) (*IntegritySigner, error) {
+	return &IntegritySigner{}, nil
+}
+
+// RegisterIntegrityFlag is a noop function that is intended to add
+// flags to `tt pack` command.
+func RegisterIntegrityFlag(cmd *cobra.Command, dst *string) {}
+
+// PackHashing is a noop method that is intended to prepare hash files
+// for applications and environment and sign them.
+func (IntegritySigner) PackHashing([]util.AppListEntry, string) error {
+	return nil
+}
```

Comment:
Pack or Sign?
```suggestion
func (Packer) Pack(paht string, []util.AppListEntry) error {
	return nil
}
```
```suggestion
func (Signer) Sign(path string, apps []util.AppListEntry) error {
	return nil
}
```
What is the second argument actually?

Reply:
I thought it this over, and I feel like neither fit the purpose nicely. I guess something like `AddIntegrityHashes` fits the function's purpose better

Reply:
I suggest you think more about universal naming.

Reply:
I gave it a thought, and I'm sold on `Sign`.